frontend/src/styles/HomePage.css

```diff
@@ -0,0 +1,98 @@
+/* Height of the logo */
+#logo{
+  height: 50px;
+}
+
+/* This defines the height of the header */
+.MuiToolbar-root.MuiToolbar-regular.MuiToolbar-gutters{
+  height: 64px;
+}
+
+/* This helps to style the home page */
+.homePage{
+  height: auto;
+}
+
+/* This is to adjust the header on the home page*/
```

Comment:
This is pedantic, but I think the "this is to" at the start of each comment is redundant. I think a better convention for us might be to simply write, "Adjusts the header on the home page" instead.